sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/AnalyzeSentimentAsyncClient.java

```diff
@@ -224,32 +232,32 @@ private AnalyzeSentimentResult convertToAnalyzeSentimentResult(DocumentSentiment
     /*
      * Transform SentenceSentiment's opinion mining to output that user can use.
      */
-    private IterableStream<MinedOpinion> toMinedOpinionList(
-        com.azure.ai.textanalytics.implementation.models.SentenceSentiment sentenceSentiment) {
+    private IterableStream<MinedOpinions> toMinedOpinionList(
+        com.azure.ai.textanalytics.implementation.models.SentenceSentiment sentenceSentiment,
+        List<DocumentSentiment> documentSentimentList) {
         // If include opinion mining indicator is false, the service return null for the aspect list.
-        if (sentenceSentiment.getAspects() == null) {
+        final List<SentenceAspect> sentenceAspects = sentenceSentiment.getAspects();
+        if (sentenceAspects == null) {
```

Comment:
is this a common pattern in Java?
At least in .NET, we always return an empty list instead of null (applies to other libraries too)

Reply:
It depends. I checked with @srnagar. Returns null is different than return an empty list. A list is null means the value is never set, a empty list means the value was set but it was empty.  "returning empty list might make the user feel like we still went ahead and mined the opinions and there were just no opinions in the text".